### DISCUSSION TITLE
Features I miss with filters

### DISCUSSION THREAD

**Discussion by outline4** [craftcms/cms#11187] [Ideas] [answered]
* Filters should only show relevant fields.
* Filters should be savable, reordable, duplicatable etc. Something like Zenbu from EE.

That list is shorter than I thought ;)

**Comment by brandonkelly** [comment, marked as answer]
I found a way to reduce the custom field condition rules to only those that are relevant for the selected source (0090ec20c393d37d01a1df69eba6d6bee6809a07). That change will come in 4.1.

Please post a new discussion for each additional suggestion.

**Reply by brandonkelly** [reply]
Craft 4.1 is out with that change ✨
